lxqtapplication.cpp

```diff
@@ -142,7 +142,7 @@ namespace
     QScopedPointer<SignalHandler> SignalHandler::instance;
 }
 
-void Application::listenToUnixSignals(QList<int> const & signoList)
+void Application::listenToUnixSignals(QList<int> const & signalList)
```

Comment:
Also, in the body of the function (this cannot be compiled).

Reply:
Whoops :/

Reply:
Btw. this wasn't any typo, but intentional name (signal numbers list) :-)

Reply:
> Btw. this wasn't any typo

The comment of the functions said `... defined in \param signalList...`.

"signolList" wasn't indicative of a number either. I think @isf63 was right to treat it as a typo, although a harmless one.

Reply:
I'll change the identifier and comment to `signoList` as that is used elsewhere in lxqtapplication.cpp/h.

Reply:
> I'll change the identifier and comment to signoList as that is used elsewhere in lxqtapplication.cpp/h.

Oh, this is getting complicated for no good reason. That method is public and has a comment; "signoList" has no meaning as a name either. Your previous change was good, and I was going to merge it after a test....

Reply:
`signoList` has a well-defined meaning - 'signo' in Unix/Linux means 'signal number'. Running `grep -o -i signo lxqtapplication.cpp` there were 13 instances of signo before the recent commit, including another signoList method param.

I agree, getting complicated, but I think it's done.

Reply:
> signoList has a well-defined meaning...

It may have been used by some devs and then copied. "Linux signo" leads to "Linux signal" in Google and Wikipedia. 

> there were 13 instances of signo

That's why I said, "That method is public and has a comment....". Anyway, I resigned from reviewing it. Hopefully, another dev will do it.